standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/Msck.java

```diff
@@ -302,7 +296,9 @@ private boolean closeTxn(String qualifiedTableName, boolean success, long txnId)
     } else {
       try {
```

Comment:
put it try only what's really needed + use abort single txn method, why all the magic with lists?

Reply:
This was already using abortTxns call before. However, changed it now. Done.